internal/resources/metal/organization_member/resource_schema.go

```diff
@@ -0,0 +1,63 @@
+package organizationmember
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+func GetResourceSchema() *schema.Schema {
+	return &schema.Schema{
```

Comment:
in terraform framework the ID must be defined. You can use https://github.com/equinix/terraform-provider-equinix/blob/1e6d5631bd42d415ba8ce0874e92b876f890b4e9/internal/resources/metal/connection/resource_schema.go#L26
You will need to add it also in the resource model and make sure the value is updated in the parse function